docs/_includes/attr-data.adoc

```diff
@@ -2,58 +2,141 @@
 Included in:
 
 - user-manual: Built-in data attributes
+
+Changes:
+Added an example/notes column; 
+deleted doctitle because it is documented elsewhere; 
+added basebackend, filetype and outdir;
+consistently use 'document' for the input file and 'output file' for the output.
+added safe-mode stuff
 ////
 
-Asciidoctor includes numerous intrinsic attributes that are assigned values when the document is rendered.
-The values of these built-in data attributes are derived from how the document is processed and when and where is it processed.
+Asciidoctor includes numerous attributes that are assigned values when the document is rendered.
+The values of these attributes are derived from how the document is processed and when and where is it processed.
```

Comment:
I propose writing this paragraph as follows:

```
Asciidoctor includes numerous attributes that get assigned values when the document is converted (i.e., intrinsic attributes).
The values of the intrinsic attributes are derived from how, when and where the document is processed (in other words, its runtime environment).
```

Note that we always want to use the term "convert" instead of "render" since render means "to display", which is the role of the browser or file viewer.